resources/css/bem/input-container.less

```diff
@@ -32,6 +34,10 @@
 
   &:focus-within {
     --input-border-colour: var(--input-border-focus-colour);
+
+    .@{top}--error& {
+      --input-border-colour: var(--input-border-error-colour);
```

Comment:
override `--input-border-focus-colour` in `--error` modifier instead? Also maybe have a lighter/darker colour than the normal error one?